config/qt5.mpb

```diff
@@ -0,0 +1,3 @@
+project: qt5_widgets {
```

Comment:
Is this mpb needed?

Reply:
No, but I'm not fond of `qt5_widgets` being the name of the base project required to use Qt5.

Reply:
Qt5Widgets is the name of the library, so using that in the mpb name makes sense to me.  Is this new for v5?  I didn't see it in v4.

Reply:
Yes, that was the major change that concerned our code, that the Gui module was split into Gui and Widgets modules.